frappe/utils/dashboard.py

```diff
@@ -118,3 +118,7 @@ def get_config(app, module):
 		return None
 	except ImportError:
 		return None
+	except Exception as e:
+		print(_("Failed to import dashboard fixtures for module {module}").format(module=module))
+		frappe.log_error(e, _("Dashboard Fixture Import Error"))
+		return None
```

Comment:
is the print required?

also, you don't need to specify None as a return, just returning should work the same.